zh_cn/README.md

```diff
@@ -0,0 +1,1627 @@
+
+# Go Concurrency Guide
+
```

Comment:
what meaning zh_cn ?

there are many projects interesting to be used like example
https://laravel-china.github.io/php-the-right-way/
https://phptherightway.com
http://phpdevenezuela.github.io/php-the-right-way/
http://wafe.github.io/php-the-right-way/